packages/mux-player/src/index.ts

```diff
@@ -290,22 +304,10 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
     return this.mediaTheme?.shadowRoot?.querySelector('media-controller');
   }
 
-  get metadataFromAttrs() {
```

Comment:
Probably low risk, but technically a breaking change. I'm fine with making the change, but we'd want to revert if there's any complaints.

Reply:
yes, I don't think this was the intention to make public and we didn't document it. sounds good